scripts/fetch_and_build_deb

```diff
@@ -49,15 +49,26 @@ fi
 # populate variables from packaging metadata file
 # shellcheck source=/dev/null
 source /buildfiles/pkgvars
+# populate pg versions to be used for packaging.
+# supported-potgres file is derived file from postgres-matrix.yml file
+# If file does not exist, skip populating file
+[[ -f "/buildfiles/supported-postgres" ]] && source /buildfiles/supported-postgres
 
-# set default values for certain packaging variables
+# Fetch pkgname, hubproj, nightlyref, versioning from pkgvars file
 declare pkglatest # to make shellcheck happy
 pkgname="${deb_pkgname:-${pkgname}}"
 hubproj="${hubproj:-${pkgname}}"
 nightlyref="${nightlyref:-master}"
-releasepg="${releasepg:-11,12,13}"
-nightlypg="${nightlypg:-${releasepg}}"
 versioning="${versioning:-simple}"
+# Fetch pg release and nightly versions from supported-postgres file which is originated from postgres-matrix.yml file
+# If this file is not found, releasepg and nightlypg parameters from pkgvars are used for defining pg versions
+pg_release_versions="${release_versions:-${releasepg}}"
+pg_nightly_versions="${nightly_versions:-${nightlypg}}"
```

Comment:
```suggestion
pg_release_versions="${release_versions:-$releasepg}"
pg_nightly_versions="${nightly_versions:-$nightlypg}"
```

Reply:
I think we should keep this since it is better using bash parameters with curly braces